docs/configuration/authentication.md

```
@@ -0,0 +1,481 @@
+  --phase both
+```
+
+## Security Best Practices
+
+1. **Never commit secrets** to version control
+2. **Use environment variables** for sensitive tokens in configuration files
+3. **Rotate tokens regularly** when testing production-like environments
+4. **Use separate credentials** for testing vs. production
+5. **Store auth configs** outside of the project directory
+6. **Set appropriate permissions** on auth config files:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)